### DISCUSSION TITLE
Gitpages error

### DISCUSSION THREAD

**Discussion by AlexPaiva** [community/community#40787] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi,

I tried adding gitpages to my git, it is stuck at 'building' for days and the deployment says it was deployed yet nothing is deployed at all.
I can't disable, enable or anything around gitpages. It is completely stuck

**Comment by airtower-luna** [comment]
If you mean the repository https://github.com/AlexPaiva/Portal-for-Service-Mesh-for-Virtualized-Environments the problem is that you have an invalid submodule in your repository at `docs/`. Remove or fix that, and it should work.

For the future please keep in mind to always link the repository/logs when asking for help, it makes it much easier for people to help. :slightly_smiling_face:

**Comment by AlexPaiva** [comment]
How do I remove it?

**Reply by AlexPaiva** [reply]
Not being the creator*

**Comment by airtower-luna** [comment, marked as answer]
What do you mean by not being the creator? The repository is yours.

The problem appears to be that the `docs` submodule doesn't point at a valid repository. What do you actually want to be there?

**Reply by AlexPaiva** [reply]
I have 4 collaborators, he added that and I have no idea how. Neither does he.
I have no idea how to remove this, I can't open it, I can't do nothing with it, I was nothing, I wanted to create a git pages page and it said to point to the docs folder

**Reply by AlexPaiva** [reply]
I want*

**Reply by AlexPaiva** [reply]
I think we managed to delete the folder but Gitpages is still stuck on "your page is being built from the root /" for 3 days now

**Reply by airtower-luna** [reply]
Where do you see anything stuck? The deployment history shows a successful deployment now:
https://github.com/AlexPaiva/Portal-for-Service-Mesh-for-Virtualized-Environments/deployments/activity_log?environment=github-pages

**Reply by AlexPaiva** [reply]
Yep, deleted everything and foced a gitpages re-build, worked. Reading the read-me.
I want gitpages to load from a folder. The only option is the "docs"  folder correct? how do I load from that?

**Reply by AlexPaiva** [reply]
Also, thanks for your help, it helped a lot!

**Reply by airtower-luna** [reply]
> I want gitpages to load from a folder. The only option is the "docs" folder correct? how do I load from that?

It's the only option if you don't want to set up your own deployment workflow. See the [Publishing from a branch](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-from-a-branch) documentation for how to do that, you can select the folder in step 7 as described there.

Before enabling that you should move the files within your repository so what you want to publish is actually in a `docs/` folder.

**Reply by AlexPaiva** [reply]
Thanks, will take a look.
